devops/helm/crunchy-postgres/value-0c27fb-prod.yaml

```diff
@@ -24,7 +24,7 @@ instances:
   name: ha # high availability
   replicas: 3
   dataVolumeClaimSpec:
-    storage: 21Gi
+    storage: 5Gi
```

Comment:
PVC for prod is kept as same as non prod environments. I believe we have a plan to update it before we move production.

Reply:
Yes